src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate: function(callback, value) {
+      this.result = callback(value, this.result);
+
+      return this;
+    },
+
+    add: function(value, result) {
+      return result + value;
+    },
```

Comment:
![image](https://user-images.githubusercontent.com/97547824/207670297-42aba134-33ff-4a11-9ff2-b5cf8d4567d8.png)
Fix it everywhere